weblate/trans/util.py

```diff
@@ -333,6 +337,23 @@ def is_unused_string(string: str):
     return string.startswith("<unused singular")
 
 
-def count_words(string: str):
+def count_words(string: str, lang_code=""):
     """Count number of words in a string."""
-    return sum(len(s.split()) for s in split_plural(string) if not is_unused_string(s))
+    if is_ngram_code(lang_code):
+        count = 0
+        for s in split_plural(string):
+            if is_unused_string(s):
+                continue
+            even = True
+            for sec in CJK_PATTERN.split(string):
+                if even:
+                    count += len(sec.split())
+                else:
+                    count += len(sec)
+                even = not even
+        return count
+    else:
+        return sum(len(s.split()) for s in split_plural(string) if not is_unused_string(s))
```

Comment:
Address RET505 error:

```suggestion
    return sum(len(s.split()) for s in split_plural(string) if not is_unused_string(s))
```